test/functional/apm_tests.js

```diff
@@ -10,6 +10,13 @@ describe('APM', function() {
     return setupDatabase(this.configuration);
   });
 
+  afterEach(function() {
+    if (this.listener) {
+      this.listener.uninstrument();
```

Comment:
What if its already uninstrumented? Presumably some cases here don't fail

Reply:
I believe i tested this, and calling `uninstrument` multiple times does not die IIRC